### PR TITLE
Chg: switch to a single servie and disable ingress usage by default

### PR DESCRIPTION
**Pull request by dunaev-opsfleet**

### CODE REVIEW

**Comment by dunaev-opsfleet** [comment]
I've decided to switch to a single service and LoadBalancer type as this way we can have a single NLB exposed with correct certificates, so that the whole application could be exposed with a single FQDN, but different certificates including Amazon controlled ones

**Comment by dunaev-opsfleet** [comment]
> Do multiple 'ports' definitions in the same service work with Helm values files?

Yes it is

File: helm/values.yaml [hunk 1]
```diff
@@ -212,14 +210,18 @@ config: |
   #
   # NLB Support
```
Comment (brennerm) [inline helm/values.yaml:211]: Should this be renamed as well?

**Review by brennerm — APPROVED** [review]

File: helm/values.yaml (hunk 1 above)
Comment (dunaev-opsfleet) [reply helm/values.yaml:211]: Well technically it's still the same in https://github.com/Telecominfraproject/wlan-cloud-ucentralgw/blob/master/ucentral.properties#L81 , so I'm not sure if we should change it here

**Review by oblom0v — APPROVED** [review]

**Pull request merged**